src/config.zig

```diff
@@ -204,7 +204,7 @@ pub const configs = struct {
             .direct_io = true,
             .direct_io_required = true,
             .cache_accounts_size_default = @sizeOf(vsr.tigerbeetle.Account) * 1024 * 1024,
-            .cache_transfers_size_default = 0,
+            .cache_transfers_size_default = @sizeOf(vsr.tigerbeetle.Transfer) * 2048,
```

Comment:
> Is 0 still a legal value here?

No - it's not currently, since the CacheMap needs the SAC. I could make a boolean disabling the SAC layer at `comptime` so that it falls straight through to the map for the `Transfer` case?

> Or wait, I guess we need enough cache to accommodate prefetch objects.. 🤔

This is handled separately by the CacheMap's `map_value_count_max`.

Reply:
Hmm, we may as well leave it for now -- maybe just add a TODO to benchmark whether disabling cachemap for transfers improves performance enough to warrant the ability to disable the SAC.